config/services.yaml

```diff
@@ -333,7 +333,12 @@ services:
         public: false
         tags:
             - { name: security.voter }
-
+    commsy.security.date_voter:
+        class: App\Security\Authorization\Voter\DateVoter
+        arguments: [ '@commsy_legacy.environment' ]
+        public: true
+        tags:
+            - { name: security.voter }
```

Comment:
Please remove the service definition, I don't think it is necessary.

Reply:
@EofChris please, check the changes.Comments resolved.